ci-operator/jobs/openshift/cluster-api-provider-ibmcloud/OWNERS

```diff
@@ -0,0 +1,18 @@
+# DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+- alexander-demichev
+- danil-grigorev
+- elmiko
+- enxebre
+- joelspeed
+- michaelgugino
+options: {}
+reviewers:
+- alexander-demichev
+- danil-grigorev
+- elmiko
+- enxebre
+- joelspeed
+- michaelgugino
```

Comment:
we should remove `danil-grigorev`, `enxebre`, and `michaelgugino` from these lists, and add `lobziik`.